### ISSUE TITLE
Text customization options

### ISSUE THREAD

**Issue by erinlynmclean** [Lattyware/massivedecks#211]
Hi! I would like to be able to make this into an educational game, and I'm looking to add explanations of some terms on the cards themselves. Since there's already the ability to customize the text, can you also add a feature to customize the font size?

I've attached an example of what I'm looking for in a final product here: 
![Picture1](https://user-images.githubusercontent.com/60625246/102923040-e0350200-4443-11eb-9968-cef9bce58164.png)

Thanks!

**Comment by Lattyware** [comment]
Yeah, adding some kind of font size styling alongside the current emphasis option shouldn't be too hard to implement. On the list.

**Comment by erinlynmclean** [comment]
Wonderful! Any idea when the feature might be implemented?
